cmd/node/node.go

```diff
@@ -13,6 +13,7 @@ import (
 	"path/filepath"
 	"runtime"
 	"runtime/debug"
+	gsync "sync"
```

Comment:
This is a nitpick, but I'd rather you "import as" our sync library than the standard sync lib, for readability. Call it "syncer" maybe. `gsync` sounds like something google-related!